Update extension.neon

convenient opened the pull request:
Fix phpstan.neon to work with latest phpstan

```
⚠️  You're using a deprecated config option autoload_files. ⚠️️
You might not need it anymore - try removing it from your
configuration file and run PHPStan again.
If the analysis fails, there are now two distinct options
to choose from to replace autoload_files:
1) scanFiles - PHPStan will scan those for classes and functions
   definitions. PHPStan will not execute those files.
2) bootstrapFiles - PHPStan will execute these files to prepare
   the PHP runtime environment for the analysis.
Read more about this in PHPStan's documentation:
https://phpstan.org/user-guide/discovering-symbols
```

convenient committed:
Update extension.neon

Fix phpstan.neon to work with latest phpstan

```
⚠️  You're using a deprecated config option autoload_files. ⚠️️
You might not need it anymore - try removing it from your
configuration file and run PHPStan again.
If the analysis fails, there are now two distinct options
to choose from to replace autoload_files:
1) scanFiles - PHPStan will scan those for classes and functions
   definitions. PHPStan will not execute those files.
2) bootstrapFiles - PHPStan will execute these files to prepare
   the PHP runtime environment for the analysis.
Read more about this in PHPStan's documentation:
https://phpstan.org/user-guide/discovering-symbols
```

convenient commented:
hello @fooman 

I am just wondering if you are able to have a look at this some point? Or is it best to look at an alternative/fork it?

Thanks

fooman commented:
@convenient thanks for the ping. Happy to go ahead with merging this and tag a new version. Do you mind confirming if https://github.com/fooman/phpstan-magento2-magic-methods/blob/master/composer.json#L13 needs bumping up as well?

convenient committed:
Update composer.json

https://github.com/bitExpert/phpstan-magento/issues/40#issuecomment-644011899

> Seems like the autoload_files option got deprecated with PHPStan 0.12.26. Removing autoload_files won't work as the file contains the logic to register the custom autoloaders

convenient commented:
@fooman https://github.com/bitExpert/phpstan-magento/issues/40#issuecomment-644011899
> Seems like the autoload_files option got deprecated with PHPStan 0.12.26. Removing autoload_files won't work as the file contains the logic to register the custom autoloaders

Updated composer.json to require 0.12.26 and above 👍

fooman commented:
Thanks Luke
